instagram/config/initializers/devise.rb

```diff
@@ -0,0 +1,265 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '3572aebe7cb811f9ce7fd2246d96d7972668022af481e04a003a1fc4f74b79291177dd768512052bb780af908d69626c656b220d03fbb2f45e8d76a64695866e'
```

Comment:
Metrics/LineLength: Line is too long. [154/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)